inc/namespace.php

```diff
@@ -180,10 +180,10 @@ function text_settings_field( array $args ) {
 		'description' => '',
 	 ] );
 
-	printf( '<input type="text" id="%1$s" name="%1$s" value="%2$s" />%3$s',
+	printf( '<input type="text" id="%1$s" name="%1$s" value="%2$s" aria-describedby="%1$s-description" class="regular-text ltr" /><p class="description" id="%1$s-description">%3$s</p>',
 		esc_attr( $args['name'] ),
 		esc_attr( $args['value'] ),
-		$args['description'] ? '<br /> <span class="description">' . esc_html( $args['description'] ) . '</span>' : ''
+		esc_html( $args['description'] )
```

Comment:
Thanks for this! I guess core must have updated the markup for those fields at some point.

This will always output a paragraph tag even there's no description hence the logic that was here originally, can you put that back?

Reply:
I figured as much about core.

I reverted my changes for the description so the only things in this PR are the two CSS classes.